tests/test_buffer.py

```diff
@@ -836,6 +836,28 @@ def test_SimpleArrayPlex_properties(self):
         self.assertEqual(len(sarr), 2 * 3 * 4)  # number of elements
         self.assertEqual(sarr.nbody, 2 - 0)
         self.assertEqual(sarr.has_ghost, False)
+        self.assertEqual(sarr.nghost, 0)
+        
+    def test_minmaxsum(self):
```

Comment:
borrow from `SimpleArrayTC`, just to make sure the methods can work.

Reply:
It is absolutely the right thing to do to copy (testing) code here.